patches/chrome-elevation_service-BUILD.gn.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/elevation_service/BUILD.gn b/chrome/elevation_service/BUILD.gn
+index 6ae8694531550df2835f3afbe5a699403f7495fb..80d31ffb5b82571a684a8bc80e40bfebf84512f9 100644
+--- a/chrome/elevation_service/BUILD.gn
++++ b/chrome/elevation_service/BUILD.gn
+@@ -9,7 +9,7 @@ import("//chrome/process_version_rc_template.gni")
+ import("//testing/test.gni")
+ 
+ midl("elevation_service_idl") {
+-  sources = [ "elevation_service_idl.idl" ]
++  sources = [ "//brave/chromium_src/chrome/elevation_service/elevation_service_idl.idl" ]
```

Comment:
do not modify lines in patches unless it's absolutely necessary. Here you can do `sources -= [ "elevation_service_idl.idl" ] sources += [ ... ]`

Reply:
Fixed